src/content/en/fundamentals/design-and-ui/_toc.yaml

```diff
@@ -15,6 +15,8 @@ toc:
         path: /web/fundamentals/design-and-ui/media/video
       - title: Multi-Device Content
         path: /web/fundamentals/design-and-ui/responsive/content
+    - title: Basics of UX
+      path: /web/fundamentals/design-and-ui/ux-basics
```

Comment:
Should read: `/web/fundamentals/design-and-ui/ux-basics/` with the trailing backslash, that should also fix the left hand nav not expanding.